equipmanager.lic

```diff
@@ -142,16 +146,12 @@ class EquipmentManager
       echo('failed to match an item, try turning on debugging with ;e UserVars.equipmanager_debug = true')
       return false
     end
-    case bput("get my #{item.short_name}", item.short_regex, '^You get ', '^What were you referring')
-    when item.short_regex
+    if get_item?(item)
```

Comment:
Rather than assume "get my <item>" will work, now use the `get_item?` helper function that respects the `tie_to`, `wield`, and `container` options so knows when to do "untie my <item>".